ambry-messageformat/src/main/java/com.github.ambry.messageformat/DeleteMessageFormatInputStream.java

```diff
@@ -55,4 +55,22 @@ public DeleteMessageFormatInputStream(StoreKey key, short accountId, short conta
     messageLength = buffer.capacity();
     buffer.flip();
   }
+
+  public DeleteMessageFormatInputStream(StoreKey key, short accountId, short containerId, long deletionTimeMs,
```

Comment:
The reason I create a different constructor is because of dependency issue.
The new constructor requires lifeVersion to be not -1, which is a constant from IndexValue.Life_Version_From_Frontend. And I don't want to create a new constant here and I can't import the IndexValue constant either. That's why I create a new constructor and leave if-else statement in the ambry-store package, where IndexValue constant can be used.